CHANGELOG.md

```diff
@@ -8,11 +8,19 @@ Subheadings to categorize changes are `added, changed, deprecated, removed, fixe
 
 # Changelog
 
-The latest published Parley release is [0.3.0](#030---2025-02-27) which was released on 2025-02-27.
-You can find its changes [documented below](#030---2025-02-27).
+The latest published Parley release is [0.4.0](#040---2025-05-08) which was released on 2025-05-08.
+You can find its changes [documented below](#040---2025-05-08).
```

Comment:
Hmm. This should generally be in the PR actually making the release, so that the changelog is consistent with the actual state of things (except for the 5 minutes between merge completing and release happening)